test/.eslintrc

```diff
@@ -0,0 +1,6 @@
+extends:
+  - '../.eslintrc'
+  - 'homeoffice/config/testing'
+
+env:
```

Comment:
env isn't needed if we are extending ../.eslintrc if it is defined there - which I assume it is as silly to have tests in es6 but src in es5.

Actually, why is this file even here? Does dotfiles not include test/.eslintrc @JoeChapman ?